migrations/909-website-reviewers.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO groups (id, name, rules, created, modified, notes)
+VALUES (50080, 'Website Reviewers', 'Websites:Review', NOW(), NOW(), 'Users who can review websites');
+UPDATE groups SET rules=CONCAT(rules, 'Websites:Review') WHERE name = 'Staff';
```

Comment:
There also should have been a prepending comma in front of `Websites:Review`; this will have broken the Staff group for anyone for whom this migration successfully ran, setting `rules` to `Apps:*,Users:Edit,Stats:View,AdminTools:View,AccountLookup:View,AppLookup:View,Lookup:View,Stats:ViewWebsites:Review`.